lib/core/datatable/components/datatable/datatable.component.ts

```diff
@@ -535,6 +535,10 @@ export class DataTableComponent implements AfterContentInit, OnChanges, DoCheck,
         return value ? 'ICONS.' + value.substring(value.lastIndexOf('/') + 1).replace(/\.[a-z]+/, '') : '';
     }
 
+    iconType(value: string): string {
+        return this.iconAltTextKey(value).split('_').pop();
```

Comment:
why we are calling the `iconAltTextKey` method instead of using 
`value.substring(value.lastIndexOf('/') + 1).replace(/\.[a-z]+/, '') : '';`